petabvis/bar_row.py

```diff
@@ -63,6 +63,11 @@ def get_sd(self):
         return sd
 
     def get_replicate_sd(self):
+        """
+        Return the standard deviation for each replicate.
+        Returns:
```

Comment:
```suggestion
        Return the standard deviation for each replicate.
        
        Returns:
```

Include a blank line after the one-line description. The first paragraph (1st line, ideally) will be used by many documentation generators as a short description.